README.md

```diff
@@ -22,13 +22,44 @@ Once created don't forget to update the:
 - Jetpack Compose setup ready to use. 
 - Sample Espresso, Instrumentation & JUnit tests.
 - 100% Gradle Kotlin DSL setup.
+- Pre-commit script to ensure gradle `check` task passes before every commit
 - CI Setup with GitHub Actions.
 - Publish to **Maven Central** with Github Actions.
 - Dependency versions managed via `buildSrc`.
 - Kotlin Static Analysis via `detekt` and `ktlint`.
 - Issues Template (bug report + feature request).
 - Pull Request Template.
 
+## Known issues
```

Comment:
I would also remove this section entirely as I don't think it's useful

Reply:
When I was trying to get started with this template I was caught out by both of the issues I've listed here. I know these aren't technically issues with the template itself, but they are issues which someone using the template might trip over. How strongly do you feel about removing these? I think it would be helpful to leave them in.

Reply:
How about we create a `KNOWN_ISSUES.md` file and we link it in the README?

Reply:
Done